general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -316,6 +317,24 @@ def supercheq(
         }
         return self.post_request("/supercheq", json_dict)
 
+    def target_info(
```

Comment:
this will need its own test in `superstaq_client_test` to pass the standalone coverage check

Reply:
(also just noticed this bug as a result of this: #486)